appveyor.yml

```diff
@@ -82,6 +82,6 @@ for:
   - mkdir build\lib
   - cp build.luarocks\deps\libuv\Release\uv_a.lib build\lib\uv.lib
   - cp -a deps\libuv\include build
-  - ps: luarocks make rockspecs\$(ls rockspecs) LIBUV_DIR=build LUA_COMPAT53_INCDIR=deps/lua-compat-5.3/c-api CFLAGS="/nologo /MT /O2"
+  - ps: luarocks make rockspecs\$(ls rockspecs) LIBUV_DIR=build LUA_COMPAT53_INCDIR=deps/lua-compat-5.3 CFLAGS="/nologo /MT /O2"
```

Comment:
This should probably still be `LUA_COMPAT53_INCDIR=deps/lua-compat-5.3/c-api`.

`LUA_COMPAT53_INCDIR` passed to LuaRocks is the folder that contains the `.h` file itself. `LUA_COMPAT53_DIR` passed to CMake is the `lua-compat-5.3` folder with `c-api` inside it.

If your goal is full backwards compatibility, I think we might need to revert #420 completely.

Reply:
My mistake, I'll try to fix.